resources/sdk/purecloudpython/templates/README.mustache

```diff
@@ -111,6 +111,82 @@ PureCloudPlatformClientV2.configuration.proxy_password = 'YOUR_PROXY_PASSWORD'
 
 The Python SDK uses `urllib3.ProxyManager` to make requests when `proxy` is given.
 
+#### SDK Logging
+
+Logging of API requests and responses can be controlled by a number of parameters on the `configuration`'s `logging` instance.
+
+`PureCloudPlatformClientV2.logger.LogLevel` values:
+1. LTrace (HTTP Method, URL, Request Body, HTTP Status Code, Request Headers, Response Headers)
```

Comment:
Why are we putting an L in front of the method names?  Will this be consistent across all of the SDKs in terms of method names.

Reply:
Just in front on enum names. I've applied that consistently across all SDKs